build.gradle

```diff
@@ -53,3 +54,16 @@ configurations.all {
     }
 }
 
+spotless {
```

Comment:
It is not a good idea to add an autoformatter to an open PR. I will not approve such a PR under any circumstances.

The reason is that an autoformatter makes many changes to the code (to fix the formatting). It is not a good use of time to read through all of these changes carefully. Therefore, there must not be any non-formatting changes mixed in with the formatting changes, so that the PR with the formatting changes can be confidently approved without having to read the changes themselves too carefully. So, adding an autoformatter must happen on an entirely separate branch, with no other changes.

Reply:
@thoriumrobot  [a1a51eb](https://github.com/thoriumrobot/VGRTool/pull/1/commits/a1a51ebd4556f4db1d1bb123d1e5da7cbdaef6e4) did not resolve this problem, because it just removed the logic to _run_ Spotless, but not the changes that the autoformatter made. The only way to get rid of those is to revert 01e566b.